lib/Driver/ToolChains.cpp

```diff
@@ -1653,10 +1621,12 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
     Arguments.push_back(context.Args.MakeArgString(SharedRuntimeLibPath));
   }
 
-  auto PreInputObjectPath = getPreInputObjectPath(ObjectLibPath);
-  if (!PreInputObjectPath.empty()) {
-    Arguments.push_back(context.Args.MakeArgString(PreInputObjectPath));
-  }
+  SmallString<128> srtPath = SharedRuntimeLibPath;
+  llvm::sys::path::append(srtPath,
+                          swift::getMajorArchitectureName(getTriple()));
+  llvm::sys::path::append(srtPath, "srt.o");
```

Comment:
Is there a more descriptive name we can use than `srt`?

Reply:
"swiftrt.o" would be fine.

Reply:
Or even just "swiftRuntime.o", we don't have to pay for bandwidth by the bit anymore!

Reply:
I'm not tied to the name.  `swiftrt.o` sounds good to me.

Reply:
As an outsider I wouldn't have immediately guessed that srt/swiftrt concerns the runtime (even if it seems obvious in hindsight). There is value in spelling things out and the cost is low.